package/yast2-caasp.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-caasp
-Version:        4.2.0
+Version:        4.3.0
```

Comment:
Any reason for 4.3.0? All YaST packages targeted for Leap 15.2 or SLE15-SP2 will have 4.2.x, unless there is a reason, please, keep consistent.

Reply:
The package will only be used in Tumbleweed (not Leap, not SLE15). I used 4.3.0 as this are major new functionality and not only some bug fixes